packages/slate-react/src/components/editor.js

```diff
@@ -153,260 +140,93 @@ class Editor extends React.Component {
 
   render() {
     debug('render', this)
+    const { controller, tmp } = this
     const props = { ...this.props }
-    const tree = this.stack.render('renderEditor', props, this)
-    return tree
+
+    // Update the props on the controller before rendering.
+    const { options, readOnly } = props
+    const plugins = this.resolvePlugins(props.plugins, props.schema)
+    const value = tmp.change ? tmp.change.value : props.value
```

Comment:
The editor will only rerender if there is a props change, should we only render prop.value here instead? Or maybe `tmp.change` should be `state.change` if we want a render cycle to happen.

Reply:
@ericedem not sure I understand what you mean. I believe here `tmp.change` should only be present if we needed to normalize the `value` before the component was mounted and ready to receive changes. So it's just saying that if there's an interim value, use it, since it will soon be flushed via `onChange`.

Reply:
Op! Sorry, I just got what you mean now. You're right, since we're already rendering via `editor.value` we're already achieving this, and the `tmp.change` logic here isn't doing anything!